docs/host-and-deploy/index.md

```diff
@@ -0,0 +1,175 @@
+   1. Navigate to the [URL Rewrite Module downloads page](https://www.iis.net/downloads/microsoft/url-rewrite#additionalDownloads). For English, select **WebPI** to download the WebPI installer. For other languages, select the appropriate architecture for the server (x86/x64) to download the installer.
+   1. Run the installer. Select the **Install** button and accept the license terms. A restart isn't required after the install completes.
+1. Open the **Internet Information Services (IIS) Manager** by executing **inetmgr.exe** from a run command prompt or by searching for **IIS** and selecting the app's link in the search results.
+1. In the **Connections** sidebar (left side of the window), open the server's node. Right-click the **Sites** folder and select **Add Website** from the menu.
+1. Configure the website:
+   1. Provide a **Site name**.
+   1. Set the **Physical path** to the *publish* folder of the deployed Blazor app. The *publish* folder contains:
+      * The *web.config* file that IIS uses to configure the website, including required redirect rules and file content types.
+      * The app's folder, *MyBlazorApp*.
+   1. Configure the **Binding**. Provide a high numbered port (for example, 8000) and leave the **Host name** blank or set it to `localhost`.
+   1. Select **OK** to add the website.
+1. Select the website in the **Sites** folder.
+1. Navigate to the Blazor app: In the IIS Manager's **Action** sidebar (right side of the window), select the link under **Browse Website** to open the app in the default browser. If the port was set to 8000, the Blazor app is available at `http://localhost:8000`.
+
+### Nginx
+
+The following *nginx.conf* file is simplified to show how to configure Nginx to send the *Index.html* file whenever it cannot find a corresponding file on disk.
+
+```
+events { }
+http {
+    server {
+        listen 80;
+
+        location / {
+            root /usr/share/nginx/html;
+            try_files $uri $uri/ /Index.html =404;
+        }
+    }
+}
+```
+
+For more information on production Nginx web server configuration, see [Creating NGINX Plus and NGINX Configuration Files](https://docs.nginx.com/nginx/admin-guide/basic-functionality/managing-configuration-files/).
+
+### Nginx in Docker
+
+Add one line to the Dockerfile, as shown in the following example:
+
+```
+FROM nginx:alpine
+COPY ./bin/Release/netstandard2.0/publish /usr/share/nginx/html/
+COPY nginx.conf /etc/nginx/nginx.conf
+```
+
+### GitHub Pages
+
+Blazor apps can be hosted on [GitHub Pages](https://pages.github.com/). For an unsupported example approach, see [Rafael Pedicini's](http://www.rafaelpedicini.com/) [Single Page Apps for GitHub Pages](http://spa-github-pages.rafrex.com/) ([rafrex/spa-github-pages on GitHub](https://github.com/rafrex/spa-github-pages#readme)). Another example using the same approach can be seen at [blazor-demo/blazor-demo.github.io
```

Comment:
We shouldn't documented "unsupported" things. But it's OK to describe work that needs to be done and then to give example solutions. For example: "To handle URL rewrites you can add a `404.html` file with a script that handles redirecting the request to the `index.html` page. For an example implementation provided by the community see [rafrex/spa-github-pages on GitHub](https://github.com/rafrex/spa-github-pages#readme)."

There also an issue if you use a project site, instead of an org site. Project sites have a path, so I think you need to update the base tag in `index.html` from `/` to `/repo-name`.